src/ApiService/ApiService/Program.cs

```diff
@@ -103,6 +103,7 @@ public class LoggingMiddleware : IFunctionsWorkerMiddleware {
             .AddScoped<IVmssOperations, VmssOperations>()
             .AddScoped<INodeTasksOperations, NodeTasksOperations>()
             .AddScoped<INodeMessageOperations, NodeMessageOperations>()
+            .AddScoped<IOnefuzzContext, OnefuzzContext>()
```

Comment:
as a person who'd be a new contributor - this is just confusing now, since I have to add a new Scoped to here and also to context. 

Can you remove all scoped additions from here (except for log) - and move them all to the context ? So we only have one place where we need to add stuff ?

Reply:
The OnefuzzConext  is just a propertybag on top of the IserviceProvider. I does not handle initialization and scopes. To Remove these instrauctions i would have to reimplement the whole dependency injection engine.